apps/src/netsim/NetSimRemoteNodeSelectionPanel.js

```diff
@@ -137,37 +121,59 @@ NetSimRemoteNodeSelectionPanel.prototype.render = function () {
   this.addRouterButton_ = this.getBody().find('#netsim-lobby-add-router');
   this.addRouterButton_.click(this.addRouterCallback_);
 
-  this.connectButton_ = this.getBody().find('#netsim-lobby-connect');
-  this.connectButton_.click(this.connectButtonCallback_);
+  this.getBody().find('.join-button').click(this.onJoinClick_.bind(this));
+  this.getBody().find('.accept-button').click(this.onJoinClick_.bind(this));
```

Comment:
`NetSimRemoteNodeSelectionPanel.prototype.onJoinClick_` is an instance method that does some work and then calls the callback passed on initialization, which was stored as `this.joinButtonCallback_`.

It's like an ogre... I mean, an onion... I mean, a cake?